xml/System.Diagnostics.Tracing/IncrementingEventCounter.xml

```diff
@@ -34,9 +34,9 @@
         <Parameter Name="eventSource" Type="System.Diagnostics.Tracing.EventSource" />
       </Parameters>
       <Docs>
-        <param name="name">To be added.</param>
-        <param name="eventSource">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="name">The name.</param>
```

Comment:
@noahfalk, @tommcdon, @Anipik  can you please help specify what name is this referring to? Is it the name for the "event counter" object? Is it the name of the provided event source? It's not clear. Same case for the other 2 constructors for IncrementingPollingCounter and PollingCounter.

Reply:
It is the name of the event Counter object.

Reply:
@sywhang

Reply:
It's the name of EventCounter object.

Reply:
Thanks!

```suggestion
        <param name="name">The name for this <see cref="T:System.Diagnostics.Tracing.IncrementingEventCounter" /> instance.</param>
```